mimic/rest/identity_api.py

```diff
@@ -382,6 +385,8 @@ def validate_token(self, request, token_id):
                 impersonator_session.username)
 
         if token_id in get_presets["identity"]["token_fail_to_auth"]:
+            # This is returning a 401 Unauthorized message but in a 404 not_found
+            # JSON data format. Is there a reason for this? An old OpenStack bug?
```

Comment:
@glyph this is the update per https://github.com/rackerlabs/mimic/pull/648#discussion_r73444319; if there is any insight into the background I'd be happy to add it.